config/routes.rb

```diff
@@ -247,6 +250,9 @@ def external_redirect(path_prefix, target)
           end
           resources :offsite_links
         end
+        # Controller removed. Whitehall frontend no longer serves these
+        # pages however the route is needed to generate path and urlgenerate path and url
+        # helper methods.
```

Comment:
These comments are in the `/admin` part of the routes file, so don't need to be here as the admin controller still exists.

Reply:
Good point, have removed